tests/dummy/app/templates/docs/key-value-targeting.md

```diff
@@ -0,0 +1,82 @@
+# Key-Value Targeting
+
+You can use the methods provided by this addon to set up page level [Google Ad Manager key-value targeting](https://support.google.com/admanager/answer/188092?hl=en) in your application.
+
+## Path Keys
+
+To configure our path targeting values that get set on all routes use `doPathTargeting` and `clearPathTargeting` in your `routes/application.js`.
```

Comment:
include a note that this requires that the `did/willTransition` events need to bubble, i.e. sub-routes should not return false from their own `did/willTransition` events, if they implement any

Reply:
Good catch. I think I'll add `return true;` to the hooks in other examples too to help drive this point home.